packages/inline-edit/src/InlineEdit.tsx

```diff
@@ -8,25 +8,27 @@ import { getInlineEditUncontrolled } from './InlineEditUncontrolled';
 import { IInlineEditProps } from './interfaces';
 
 
-interface IInlineEditState {
-  isEditing?: boolean;
-}
-
 const getPureInlineEdit: <TFieldValue>() => FC<
```

Comment:
Can you rewrite this function as a component?

Reply:
yes, sure 👍